templates/lobby.html

```diff
@@ -739,6 +739,86 @@
     drawingBoard.onmousemove = mouseUpdate;
     drawingBoard.onmouseleave = mouseUpdate;
 
+    let sizeSlot = 0;
+        let toolSlot = 0;
+        let colorSlot = 0;
+        let colorBox = document.getElementsByClassName('color-button');
+        var keyMap = {};
+        document.body.addEventListener('keydown', function(e) {
+            keyMap[e.key] = true;
+        });
+        document.body.addEventListener('keyup', function(e) {
+            keyMap[e.key] = false;
+        });
+        
+        drawingBoard.onwheel = function wheelUpdate(e) {
+            event.preventDefault();
+
+            if (keyMap['g']) {
+                sizeSlot += Math.min(Math.max(-1, e.deltaY), 1);
+                if (sizeSlot < 0) {
+                    sizeSlot = 0;
+                } else if (sizeSlot > 3) {
+                    sizeSlot = 3;
+                }
+
+                switch (sizeSlot) {
+                    case 0:
+                        n = 8;
+                        break;
+                    case 1:
+                        n = 15;
+                        break;
+                    case 2:
+                        n = 30;
+                        break;
+                    case 3:
+                        n = 40;
+                        break;
+                    default:
+                        console.log('No existing slot?');
+                        break;
+                }
+                document.getElementById('size' + n).checked = true;
+                setLineWidth(n);
+        
+            } else if (keyMap['f']) {
+                toolSlot += Math.min(Math.max(-1, e.deltaY), 1);
```

Comment:
Why do we need this?

Reply:
Well, I'm not sure what you mean by "this", but I guess the answer is: probably because I don't know much of javascript and that's the first way I found to do that? :o